homeassistant/components/orangepi_gpio/binary_sensor.py

```diff
@@ -0,0 +1,68 @@
+"""Support for binary sensor using Orange Pi GPIO."""
+import logging
+
+from homeassistant.components import orangepi_gpio
```

Comment:
Integration package imports should always be relative.
```py
from . import setup_input, setup_mode, etc
```

Reply:
#27958